Задания с 28.09.2021/28.09.2021-Task3/28.09.2021-Task3/Phonebook.c

```diff
@@ -0,0 +1,273 @@
+﻿#pragma warning (disable: 4996 5045 4668 4710 6031)
+
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+#include <locale.h>
+
+typedef struct
+{
+    char name[50];
+    char phoneNumber[30];
+} Note;
+
+int initializeMemory(Note phoneBook[], const char fileName[])
+{
+    FILE* database = fopen(fileName, "r");
+    if (database == NULL)
+    {
+        database = fopen(fileName, "w");
+        fclose(database);
+        return 0;
+    }
+    int notesRead = 0;
+    while (!feof(database))
+    {
+        Note note = { 0 };
+        fscanf(database, "%s%s", note.name, note.phoneNumber);
+        if (note.name[0] == 0)
```

Comment:
```suggestion
        if (note.name[0] == '\0')
```
Это же значение символьного типа, хоть 0 и '\0' в памяти представляются одинаково